src/ssim.jl

```diff
@@ -62,6 +62,10 @@ struct SSIM{A<:AbstractVector} <: FullReferenceIQI
     end
 end
 
+# default values from [1]
+const SSIM_KERNEL = ImageFiltering.KernelFactors.gaussian(1.5, 11) # kernel
```

Comment:
Unfortunately, this doesn't work very well because calculating this constant value requires loading `ImageFiltering` eagerly, and breaks the latency optimization I made in #49.

BUT, we can manually calculate and paste the array result of `ImageFiltering.KernelFactors.gaussian(1.5, 11)` here. (I _think_ we'll need to make it an centered OffsetArray). It's midnight here and I am about to get some rest, can you try and check it?

Reply:
Done and added a test.